test/bin/ci_phase_test.sh

```diff
@@ -14,7 +14,10 @@ if [ ! -d "${RF_VENV}" ]; then
     "${ROOTDIR}/scripts/fetch_tools.sh" robotframework
 fi
 
-for scenario in ./scenarios/*.sh; do
+# Set up the scenario sources
+[[ -z "${PERIODICS}" ]] && SCENARIO_SOURCES="scenarios/*.sh" || SCENARIO_SOURCES="scenarios/periodics/*.sh"
```

Comment:
Instead of making this variable specific to the periodic tests, can we make it something like `SCENARIOS_DIR`? Then the default would be `scenarios` and the periodic job can override that to set it to `scenarios-periodic`. That's similar to how we manage the suites, for example, and not nesting the directories avoids having to be careful to only process files when looping over the directory contents.

Reply:
Went with this approach in the end. This is handled by env vars in common.sh.